src/Features/Context/WebContext.php

```diff
@@ -89,32 +85,32 @@ public function setGoutteDriver()
      *
      * @AfterScenario @remote
      */
-    public function resetGoutteDriver()
+    public function resetGoutteDriver(): void
     {
         $this->minkContext->getMink()->setDefaultSessionName($this->previousMinkSession);
     }
 
-    /**
-     * @return \Surfnet\SamlBundle\Entity\IdentityProvider
-     */
-    public function getIdentityProvider()
+    public function getIdentityProvider(): IdentityProvider
     {
         /** @var RequestStack $stack */
         $stack = $this->kernel->getContainer()->get('request_stack');
```

Comment:
Candidate for DI

Reply:
Yup certainly is. Looking if this is feasible here at this point